src/backend/utils/load/ag_load_labels.c

```diff
@@ -81,7 +81,8 @@ void vertex_field_cb(void *field, size_t field_len, void *data) {
         if (cr->fields == NULL) {
             cr->error = 1;
```

Comment:
Brace needs to be on its own line -

some_function_or_command(blah blah,...)
{